services/site/src/pages/privacy-policy.tsx

```diff
@@ -436,7 +436,7 @@ const PrivacyPolicy: React.FunctionComponent = () => {
             <p>
               We do not use cookies or similar tracking technologies (like web beacons and pixels) to access or store information. We integrated the
```

Comment:
Please update the "last updated" date at the beginning of this page to 2023 as we made changes now by replacing splitbee.